.github/workflows/ci-pipeline.yml

```diff
@@ -0,0 +1,14 @@
+name: CI Pipeline
+
+on:
+  workflow_dispatch:
```

Comment:
Add another workflow to run pitest every week. (The setup broke without knowing. Also it can be nice to record the mutation score somewhere.)